tests/src/Simple/PInvoke/PInvoke.cs

```diff
@@ -118,8 +120,9 @@ private static void TestSafeHandle()
 
             ThrowIfNotEquals(true, SafeHandleTest(hnd, intVal), "Ansi String marshalling failed.");
         }
+#endif        
     }
-
+#if Windows_NT
```

Comment:
Marshal.GetLastWin32Error() is not Windows specific. It maps to errno on Unix. This test should be enabled on Unix too - you just need to use method in your helper to test it, instead of Win32 API.